src/template_compiler.erl

```diff
@@ -196,7 +196,7 @@ block_lookup({ok, TplFile}, BlockMap, ExtendsStack, DebugTrace, Options, Vars, R
             case lists:member(Module, ExtendsStack) of
                 true ->
                     FileTrace = [Module:filename() | [ M:filename() || M <- ExtendsStack ]],
-                    ?LOG_ERROR("[template_compiler] Template recursion: ~p", [FileTrace]),
+                    ?LOG_ERROR(#{ text => "Template recursion.", trace => FileTrace}),
```

Comment:
Just an idea, could we transform the trace to a stack trace format?  Then the logger can show it as a stack trace.

Reply:
Probably.. I want to change the yecc errors top maps as well. That will probably improve the readability. Not sure what a filetrae looks like tbh.

Reply:
Would be great, the Yecc errors are quite confusing. We should be able to see in one go what the problem is and where.